src/Sylius/Bundle/AdminBundle/spec/Action/Account/GetRequestPasswordResetActionSpec.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\AdminBundle\Action\Account;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\AdminBundle\Form\RequestPasswordResetType;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+final class GetRequestPasswordResetActionSpec extends ObjectBehavior
```

Comment:
We do not need this spec for sure :dancer: controllers (or controllers actions) should be as simple as possible and all the functionality should be covered by Behat scenarios. _Specing_ controller can lead to justifying putting there too much logic, which we definitely want to avoid 🖖